libs/logic-apps-shared/src/designer-client-services/lib/uiInteractionsService.ts

```diff
@@ -0,0 +1,21 @@
+import type { UiInteractionData } from '../../utils/src/lib/models/uiInteractionData';
+import { AssertionErrorCode, AssertionException } from '../../utils/src';
+import type { TopLevelDropdownMenuItem } from '../../utils/src/lib/models/topLevelDropdownMenuItem';
+
+
+export interface IDesignerUiInteractionsService {
```

Comment:
where is the getNodeContextMenuItems? Do please implement the entire proposed service